templates/press-centre/index.html

```diff
@@ -5,102 +5,67 @@
 {% block meta_description %}The home of Canonical media resources, news stories and press releases.{% endblock %}
 
 {% block content %}
-  <section class="p-strip--standard-dark is-dark is-shallow">
-    <div class="row u-equal-height u-vertically-center">
-      <div class="col-7 p-card--suru">
-        <h1 class="p-heading--1">Press centre</h1>
-        <p class="p-heading--4">The home of Canonical media resources, news stories and press releases. Journalists seeking further information can contact <a class="p-link--inverted" href="mailto:pr@canonical.com">pr@canonical.com</a>
-        </p>
+
+<section class="p-section">
+  <div class="u-fixed-width">
+    <div class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
+      <ol class="p-breadcrumbs__items p-text--x-small-capitalised">
+        <li class="p-breadcrumbs__item "><a href="/company">Company</a></li>
```

Comment:
This page doesn't exist. I'd imagine if we're going to have breadcrumbs we need a landing page for this bubble, otherwise I don't think it makes sense to have them

Reply:
ok, will remove. cc @juanruitina

Reply:
done